### ISSUE TITLE
import error

### ISSUE THREAD

**Issue by NatalieZG** [myfavouritekk/TPN#2]
@myfavouritekk Thanks for sharing the code. 
When I go to step 3, I come across an error as follow:

Traceback (most recent call last):
  File "tools/propagate/sequence_roi_propagation.py", line 26, in <module>
    from fast_rcnn.craft import sequence_im_detect
ImportError: cannot import name sequence_im_detect

Then I check the fast_rcnn.craft.py and there is no such a function "sequence_im_detect", only "im_detect".

Would u please help me with this issue? THX!

**Comment by Solomon1588** [comment]
I met this import error too, there is something wrong in cloning recursively. You'd better clone the 'tubelet_proposal' branch of py-faster-rcnn, the function sequence_im_detect is in this branch's fast_rcnn.craft.py. Wish you success :D

**Comment by NatalieZG** [comment]
@Solomon1588 Thanks so much! I've worked it out with your suggestion.

**Comment by myfavouritekk** [comment]
@Solomon1588 Right. There seems to be a problem in submodule. I will have a commit to fix it. Thanks.

**Comment by samanthawyf** [comment]
Hi, I want to seek help. Could you tell me what is the proposal_file in the step2 command?
`python tools/data/box_proto_from_proposals.py proposal_file vid_root save_dir`
I download the VID_ResNet_152_4b35_RPN_proposals.zip and unzip it, and find two .mat files. I try the .mat file as proposal_file, and it goes wrong. Thanks a lot. @NatalieZG @Solomon1588
